src/modules/playgame.js

```diff
@@ -0,0 +1,33 @@
+import { postGameScore, getGameScore } from './gameScore.js';
+
+const playGame = () => {
+  const btnRefresh = document.querySelector('.refresh');
+  const btnAddScore = document.querySelector('.add_score');
+  const listScore = document.querySelector('.list_score');
+  const name = document.querySelector('.name');
+  const score = document.querySelector('.score');
+
+  const displayScores = async () => {
+    const scores = await getGameScore();
+    listScore.innerHTML = '';
+    scores.forEach((score) => {
+      listScore.innerHTML += `<li>${score.user}: ${score.score}</li>`;
+    });
+  };
+
+  const addScores = async () => {
+    const data = {
+      user: name.value,
+      score: score.value,
+    };
+    postGameScore(data);
+    name.value = '';
+    score.value = '';
```

Comment:
- [ ] I'm unable to store nor fetch data from the API. When I enter score details and submit, then later refresh the page, the table/UI does not display the entered values. Let's ensure that entered scores are stored and fetched to and from the API correctly.

[screencast-localhost_8081-2023.05.18-05_45_01.webm](https://github.com/joramot/Leaderboard_JoseRM/assets/12745474/b6f790bf-f339-4a08-a99c-b5c78fb3f63a)